src/drivers/github.go

```diff
@@ -42,9 +43,9 @@ func (d *githubCodeHostingDriver) CanMergePullRequest(branch, parentBranch strin
 }
 
 func (d *githubCodeHostingDriver) GetNewPullRequestURL(branch string, parentBranch string) string {
-	toCompare := branch
+	toCompare := url.PathEscape(branch)
 	if parentBranch != git.GetMainBranch() {
-		toCompare = parentBranch + "..." + branch
+		toCompare = url.PathEscape(parentBranch) + "..." + url.PathEscape(branch)
 	}
 	return fmt.Sprintf("%s/compare/%s?expand=1", d.GetRepositoryURL(), toCompare)
```

Comment:
Rather than calling `url.PathEscape` three times above, you could just call it here on `toCompare`.
```suggestion
	return fmt.Sprintf("%s/compare/%s?expand=1", d.GetRepositoryURL(), url.PathEscape(toCompare))
```

Reply:
👍 I originally thought that the ellipses would get encoded too.